src/audio/dai.c

```diff
@@ -218,6 +218,19 @@ static void dai_free(struct comp_dev *dev)
 	rfree(dev);
 }
 
+static int dai_comp_get_hw_params(struct comp_dev *dev,
+				  struct sof_ipc_stream_params *params)
+{
+	struct dai_data *dd = comp_get_drvdata(dev);
+
+	trace_dai("dai_hw_params()");
+
+	/* fetching hw dai stream params */
+	dai_get_hw_params(dd->dai, params);
```

Comment:
return value here for dai_get_hw_params()

Reply:
Updated.